### PR TITLE
added myself

### PR DESCRIPTION
**Pull request by popcornell**

### CODE REVIEW

File: _pages/members.md [hunk 1]
```diff
@@ -404,6 +404,15 @@ order: 1
...
+    <div class="col-sm mt-1 mt-md-1">
+        <div class="square">
+        <a href="https://github.com/popcornell">
+            <img class="img-fluid rounded z-depth-1" src="{{ site.baseurl }}/assets/img/samuele.jpg">
+        </a></div>
+        <div class="caption">
+            Samuele Cornell
+        </div>
+    </div>
 </div>
```
Comment (ftshijt) [inline _pages/members.md:416]:
```suggestion
</div>
<div class="row mt-3">
    <div class="col-sm mt-1 mt-md-1">
        <div class="square">
        <a href="https://github.com/popcornell">
            <img class="img-fluid rounded z-depth-1" src="{{ site.baseurl }}/assets/img/samuele.jpg">
        </a></div>
        <div class="caption">
            Samuele Cornell
        </div>
    </div>
     <div class="col-sm mt-3 mt-md-0">
    </div>
    <div class="col-sm mt-3 mt-md-0">
    </div>
    <div class="col-sm mt-3 mt-md-0">
    </div>
</div>
```

File: _pages/members.md [hunk 2]
```diff
@@ -421,6 +430,7 @@ order: 1
 
 ##### Visitors & Collaborators
 <ul>
+<li>2022. 05 - 2022. 09: Samuele Cornell (Università Politecnica delle Marche, Italy)</li>
```
Comment (ftshijt) [inline _pages/members.md:433]:
```suggestion
```

Comment (ftshijt) [reply _pages/members.md:433]: This is for previous visitors

**Review by ftshijt — COMMENTED** [review]
Many thanks! Some minor suggestions

File: _pages/members.md (hunk 2 above)
Comment (popcornell) [reply _pages/members.md:441]: ok thanks !

**Comment by popcornell** [comment]
Edited

**Pull request merged**